src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
This should be a class, not just a collection of functions in a module. A class will let you manage this easier.

Reply:
created a class